rainforest-cli.go

```diff
@@ -236,10 +237,10 @@ func main() {
 		},
 		{
 			Name:        "rm",
-			Usage:       "Remove an RFML test locally and remotely",
+			Usage:       "Remove an RFML test locally and remotely (if token is specified)",
```

Comment:
Good point, I don't know why it somehow slipped my mind, that you can actually remove file yourself... lol...